app-scientific/vtk/autobuild/defines

```diff
@@ -4,9 +4,9 @@ PKGDES="The Visualization Toolkit, a software for manipulating and displaying sc
 PKGDEP="boost cgns double-conversion eigen-3 expat ffmpeg fmt freetype gdal \
         gl2ps glew hdf5 jsoncpp lapack libarchive libglvnd libharu \
         libjpeg-turbo libogg libpng libtheora libtiff libxml2 lz4 mesa netcdf \
-        openslide postgresql proj pugixml python-3 qt-5 sqlite utfcpp x11-lib \
-        zlib"
-BUILDDEP="doxygen gnuplot graphviz motif pyqt5 tcl tk"
+        openslide postgresql proj pugixml python-3 qt-5 sqlite utfcpp verdict \
+        x11-lib zlib"
```

Comment:
QA in build products: libraries linked but not declared as dependencies: `mariadb xz`. Declare as needed.

QA in build products: libraries not appearing in dynamic section but listed as dependencies: `libharu ffmpeg utfcpp lapack` Verify these libraries are actually used.